forms-flow-api/src/formsflow_api/services/form_process_mapper.py

```diff
@@ -180,3 +206,16 @@ def unpublish_previous_mapper(mapper_data: dict) -> None:
 
         except Exception as err:
             raise err
+
+    @staticmethod
+    @user_context
+    def check_tenant_authentication(mapper_id: int, **kwargs) -> int:
```

Comment:
Could you rename this to check_tenant_authorization instead of authentication ? As authentication means logging in process and not checking authorization.

Reply:
good point, will change